src/server/game/Handlers/MovementHandler.cpp

```diff
@@ -325,11 +325,14 @@ void WorldSession::HandleMovementOpcodes(WorldPacket & recvData)
 
     recvData.rfinish();                         // prevent warnings spam
 
-    // pussywizard: typical check for incomming movement packets
-    if (!mover || !mover->IsInWorld() || mover->IsDuringRemoveFromWorld() || guid != mover->GetGUID())
+    // prevent tampered movement data
+    if (!guid || guid != mover->GetGUID()) {
         return;
```

Comment:
add `recvData.rfinish();                     // prevent warnings spam` before `return`

Reply:
done, thank you :heart: